plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/security/realms/ldap/internal/connector/dao/DynamicGroupsIT.java

```diff
@@ -47,7 +49,7 @@ public void testUserManagerWithDynamicGroups()
     // env.put( Context.PROVIDER_URL, "o=sonatype" );
```

Comment:
Is possible, but these lines were present in IT from before this PR, and I did care for "general cleanup". Will remove those.